src/components/Home/Home.js

```diff
@@ -241,13 +234,13 @@ const Home = (props) => {
           </div>
           <div className={classes.integrationText}>
             <Typography variant="h6" className={classes.integrationTitle}>
-              consectetur adipiscing
+              Reward players
             </Typography>
             <Typography
               variant="subtitle1"
               className={classes.integrationSubTitle}
             >
-              Excepteur sint occaecat cupidatat non proident
+              When there will be winners, they will be rewarded with the pool's generated interest 
```

Comment:
maybe more like "The winners will be rewarded..."
and "All players will get their money back" or something - **this is important**